sojida/iReporter
# #162444253 update middleware

#### What does this PR do?
- updates the middleware functions

#### Description of Task to be completed?
- add test for middleware functions


#### How should this be manually tested?
- clone the repository
- check out into the branch `ch-update-middware-function-162444253`
- install dependencies
- test api endpoints from readme with postman 

#### What are the relevant pivotal tracker stories?
- #162444253
